src/decaylanguage/dec/dec.py

```diff
@@ -96,7 +107,7 @@ def __init__(self, *filenames: Iterable[str]) -> None:
         """
         self._grammar: Optional[str] = None  # Loaded Lark grammar definition file
         self._grammar_info: Optional[
-            dict[str, Any]
+            Dict[str, Any]
```

Comment:
As long as we support Python 3.6 (which hit EOL last month, so probably not long?), we are stuck with using `Dict[str, str]` instead of `dict[str, str]` (along with `Union[str, int]` instead of `str | int`, etc.) :'(

Reply:
Thanks. Indeed I looked at most recent doc when learning and did not look at support of old versions ;-). Yes, I don't think 3.6 will need to stay for long. In fact I know of nobody still on 3.6.

Reply:
That will be nice indeed, and less to maintain.